Header header            # Contains timestamp information for the most recent update to the object list
TrackedObject[] objects  # The list of segmented objects
